tlsutil/config.go

```diff
@@ -266,6 +271,7 @@ func (c *Configurator) Update(config Config) error {
 	c.grpc = *grpc
 	c.https = *https
 	c.internalRPC = *internalRPC
+	c.autoTLS.grpcServerUseTLS = config.GRPC.UseAutoCert
```

Comment:
Do we need the `grpcServerUseTLS` field any longer? Couldn't the `GRPCServerUseTLS` method just check `c.grpc.UseAutoCert`?

Reply:
I will need to add a new property to protocolConfig in this case, I think. But I can do that, no prob.

Reply:
Pushed a new commit removing Configurator's autoTLS.grpcServerUseTLS, using a protocolConfig.useAutoCert instead.